packages/cli/package.json

```diff
@@ -47,6 +49,8 @@
     "@types/start-server-webpack-plugin": "^2.2.0",
     "@types/webpack-env": "^1.15.2",
     "@types/webpack-node-externals": "^2.5.0",
+    "@typescript-eslint/eslint-plugin": "^v3.10.1",
+    "@typescript-eslint/parser": "^v3.10.1",
```

Comment:
v4 is the latest version, but it's newly released and still seems to have some issues, or it just requires more changes. Either way I'm pushing that a bit.